ckanext/depositar_theme/templates/home/index/idx_banner.html

```diff
@@ -0,0 +1,24 @@
+<div class="idx banner">
+	<div class="container">
+		<div class="heading">
+			<div class="slogan">
+				<h2>{{ _('deposit・<wbr>discover・<wbr>reuse') }}</h2>
+			</div>
+			<div class="search">
+				<input type="text" class="input-search" placeholder="搜尋資料集...">
+				<button type="submit" class="btn-search"><i class="fa fa-search"></i></button>
+			</div>
+			<div class="function-btn">
+				<a class="btn create-dataset" href="{{ h.url_for('/dataset/new') }}">{{ _('Create Dataset') }}</a>
+				<a class="btn create-project" href="{{ h.url_for('/organization/new') }}">{{ _('Create Project') }}</a>
+				<a class="btn view-all" href="{{ h.url_for('/dataset') }}">{{ _('All Datasets') }}</a>
+			</div>
+		</div>
+		<div class="animation">
+			<div class="animation-left">
+				<img src="{{ h.url_for('/images/logo_ani.svg') }}">
```

Comment:
public assets 不需使用 `url_for()`

```suggestion
				<img src="images/logo_ani.svg">
```